params/tests.yaml

```diff
@@ -0,0 +1,22 @@
+rrna_names:
+  - "16S"
+  - "23S"
+  - "5S"
+genes_names:
+  - "dnaA"
+  - "rpoB"
+  - "recA"
+  - "gyrB"
+  - "gyrA"
+
+outdir:
+  "test_results"
+ncbi_apikey:
+  "b28a135f33ed4dc7a5dea5dcfa981db57b09"
```

Comment:
@Mxrcon , don't show this key in public! 😅 

You can read this via java/groovy methods in the `nextflow.config` file like below 

```
ncbi_apikey = System.getenv("NCBI_API_KEY")
```

Reply:
I suggest, you rebase and force push over here.

Best to change the key via NCBI portal as well.

Reply:
@.@ true, that was my fault, I'll remove my key from ncbi portal and fix it on all of the branches